test/unit/mocked_data/test_get_optics/interface_shutdown/expected_result.json

```diff
@@ -44,10 +44,10 @@
 						"min": 0.0
 					}, 
 					"input_power": {
-						"max": 0.0, 
-						"avg": 0.0, 
+						"max": -100.0, 
+						"avg": -100.0, 
 						"instant": -40.0,
```

Comment:
-40.0 is right here as that was value reported in show interface tranceiver - output.